src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -86,6 +88,20 @@ const std::vector<DataView::Column>& LiveFunctionsDataView::GetColumns() {
   return columns;
 }
 
+[[nodiscard]] static std::string BuildTypeColumnsString(
+    const orbit_client_data::ScopeInfo& scope_info) {
+  switch (scope_info.type) {
+    case orbit_client_data::ScopeType::kApiScope:
+      return "MS";
+    case orbit_client_data::ScopeType::kApiScopeAsync:
+      return "MA";
+    case orbit_client_data::ScopeType::kHookedFunction:
+      return "D";
```

Comment:
It seems that we redefine those strings (i.e., "H", "D") in several places and sometimes give them different names.

I would prefer to define them only once and use them consistently in some other places.